www/src/components/shell/terminal/NextSteps.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Button,
+  CloseIcon,
+  DiscordIcon,
+  IconFrame,
+  Modal,
+  StarIcon,
+  ThumbsUpFilledIcon,
+  ThumbsUpIcon,
+} from '@pluralsh/design-system'
+import { Flex, P, Span } from 'honorable'
+import {
+  cloneElement,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+import Fireworks from '@fireworks-js/react'
+
+import useOnboarded from '../hooks/useOnboarded'
+
+import { State, TerminalContext } from './context/terminal'
+
+function HoveredIcon({
+  icon,
+  color,
+  hoveredColor,
+  textValue,
+  ...props
+}: any) {
+  const [isHovered, setIsHovered] = useState(false)
+
+  return (
+    <IconFrame
+      cursor="pointer"
+      padding="xsmall"
+      textValue={textValue}
+      icon={cloneElement(icon, {
+        color: isHovered ? hoveredColor : color,
+      })}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
```

Comment:
As above.

Reply:
I think it was done this way to be able to override the icon color on hover, but also ignore hover when it is selected. This is also reused from existing code.

```
              <HoveredIcon
                size="small"
                icon={feedback === 'good' ? <ThumbsUpFilledIcon /> : <ThumbsUpIcon />}
                color={feedback === 'good' ? 'icon-success' : 'icon-light'}
                hoveredColor="icon-success"
                textValue="Good"
                onClick={() => setFeedback('good')}
              />
```

Reply:
Sure, it's hard to determine which parts of code are new. Looks to be more custom that I thought. Let's keep it then.